source/administration/command-line-tools.rst

```diff
@@ -942,6 +1072,22 @@ mattermost team add
 
       sudo ./mattermost team add myteam user@example.com username
 
+mattermost team archive
+~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Archive teams based on name.
```

Comment:
For a couple of the team CLI commands we reference what the `name` field refers to. Might be useful here and in the `team search` command below.

https://github.com/mattermost/docs/blob/f411eb3f36c33e27bf5e1744c39553df01fb785b/source/administration/command-line-tools.rst#mattermost-team-add

Reply:
Updated